packages/edit-site/src/components/post-list/index.js

```diff
@@ -174,7 +174,7 @@ function useView( postType ) {
 const DEFAULT_STATUSES = 'draft,future,pending,private,publish'; // All but 'trash'.
 
 function getItemId( item ) {
-	return item.id.toString();
+	return item?.id.toString();
```

Comment:
In which case, we call this with an empty "item"?

Reply:
That's right. This PR is just to stop the browser crashing.